test/datahike/test/temporal_index.cljc

```diff
@@ -0,0 +1,115 @@
+(ns datahike.test.temporal-index
+  (:require
+    #?(:cljs [cljs.test :as t :refer-macros [is are deftest testing]]
+       :clj  [clojure.test :as t :refer [is are deftest testing use-fixtures]])
+    [datahike.api :as d]
+    [datahike.db :as dd]))
+
+(def schema-tx [{:db/ident       :name
+                 :db/valueType   :db.type/string
+                 :db/unique      :db.unique/identity
+                 :db/index       true
+                 :db/cardinality :db.cardinality/one}
+                {:db/ident       :age
+                 :db/valueType   :db.type/long
+                 :db/cardinality :db.cardinality/one}
+                {:name "Alice"
+                 :age  25}
+                {:name "Bob"
+                 :age  35}])
+
+(defn config [uri]
+  {:uri        uri
+   :initial-tx schema-tx})
+
+(deftest test-base-history
+  (let [uri "datahike:mem://test-base-history"
+        _ (d/delete-database uri)
+        _ (d/create-database (config uri))
+        conn (d/connect uri)]
+
```

Comment:
The pattern `create-database` followed by `connect` is often recurring. Shouldn't we create a function `create-database-and-connect`?